Source/common/SNTConfigurator.mm

```diff
@@ -1326,9 +1330,19 @@ - (BOOL)enableAPNS {
   return [number boolValue];
 }
 
-- (BOOL)enableNATS {
+- (BOOL)enablePushNotifications {
   // TODO: Consider supporting enablement from the sync server.
-  return [self.configState[kEnableNATS] boolValue];
+  // Check new key first, then fall back to deprecated key for backward compatibility
+  NSNumber *value = self.configState[kEnablePushNotifications];
+  if (value != nil) {
+    return [value boolValue];
+  }
+  NSNumber *deprecatedValue = self.configState[kEnableNATS];
+  if (deprecatedValue != nil) {
+    return [deprecatedValue boolValue];
+  }
+  // Default to true when neither key is set, as documented
+  return YES;
```

Comment:
This seems to be a change from how `EnableNATS` worked. Old code would default to `NO` if not set:
```
- (BOOL)enableNATS {
  // TODO: Consider supporting enablement from the sync server.
  return [self.configState[kEnableNATS] boolValue];
}
```
I recall this was originally enabled by default, but we had decided to change that logic and require an active config option.

Reply:
>I recall this was originally enabled by default, but we had decided to change that logic and require an active config option

This was because of the interval handling. It would have prevented changes from applying. Since that's fixed I was making that change now, for 2025.12 / 2026.1.